sql/updates/world/6.x/2016_05_13_00_world.sql

```diff
@@ -0,0 +1,736 @@
+-- Tol Barad --
+
+-- Spawns are located half way down
+SET @CGUID := 250970; -- 214 creature entries
+SET @OGUID := 234355; -- 111 gameobject entries
```

Comment:
new range is need again :P compared with 4.3.4 db and found one commit missing on 6.x